### ISSUE TITLE
Value cannot be null. (Parameter 'joints') when exporting monkey_king.vmdl_c

### ISSUE THREAD

**Issue by xGitHubUser999x** [SteamDatabase/ValveResourceFormat#253]
Describe the bug
When I trying to export DotA2 Monkey King model to gtlf format, the following error appears:
`One or more errors occured. (Value cannot be null. (Parameter 'joints'))`

Reference file in a Valve game
Dota 2 `game/dota/pak01_dir.vpk/models/heroes/monkey_king/monkey_king.vmdl_c

Expected behavior
I expected to get the exported gtlf file.

Version info
VRF version: 0.1.3.1229
Platform: Windows 8.1

**Comment by xPaw** [comment]
Stacktrace:
```
System.AggregateException: One or more errors occurred. (Value cannot be null. (Parameter 'joints'))
 ---> System.ArgumentNullException: Value cannot be null. (Parameter 'joints')
   at SharpGLTF.Guard.MustShareLogicalParent(ModelRoot a, String aName, LogicalChildOfRoot b, String bName)
   at SharpGLTF.Schema2.Schema2Toolkit.WithSkinnedMesh(Node node, Mesh mesh, Matrix4x4 meshPoseTransform, Node[] joints)
   at GUI.Types.Exporter.GltfModelExporter.<ExportToFile>g__AddMeshNode|10_0(String name, Mesh mesh, <>c__DisplayClass10_0& ) in D:\GitHub\ValveResourceFormat\GUI\Types\Exporter\GltfModelExporter.cs:line 60
   at GUI.Types.Exporter.GltfModelExporter.ExportToFile(String resourceName, String fileName, Model model) in D:\GitHub\ValveResourceFormat\GUI\Types\Exporter\GltfModelExporter.cs:line 95
   at GUI.Types.Exporter.ExportFile.<>c__DisplayClass0_0.<Export>b__0(Object _, EventArgs __) in D:\GitHub\ValveResourceFormat\GUI\Types\Exporter\ExportFile.cs:line 73
   at GUI.Forms.GenericProgressForm.<OnShown>b__6_0() in D:\GitHub\ValveResourceFormat\GUI\Forms\GenericProgressForm.cs:line 32
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
   --- End of inner exception stack trace ---
```

**Comment by xPaw** [comment]
Fixed in d0be326d6b0095fad828f89eec9e1f9c4367f4f2
